### ISSUE TITLE
winbind.c:585:4: warning: implicit declaration of function ‘mppe_set_keys’

### ISSUE THREAD

**Issue by floppym** [paulusmack/ppp#2]
I get the following warnings when building with -Wall. This may be resolved by defining the MPPE macro when building this file (-DMPPE).

```
winbind.c: In function ‘winbind_chap_verify’:
winbind.c:585:4: warning: implicit declaration of function ‘mppe_set_keys’ [-Wimplicit-function-declaration]
    mppe_set_keys(challenge, session_key);
    ^
winbind.c:630:4: warning: implicit declaration of function ‘mppe_set_keys2’ [-Wimplicit-function-declaration]
    mppe_set_keys2(session_key, &response[MS_CHAP2_NTRESP],
    ^
```

**Comment by paulusmack** [comment]
Seems like this would be caused by a mismatch in whether MPPE is defined in pppd/Makefile.linux vs. pppd/plugins/Makefile.linux.

**Comment by floppym** [comment]
It seems you fixed this in 2b05e22c62095e97dd0a97e4b5588402c2185071.
